projects/packages/my-jetpack/_inc/components/stats-section/count-comparison-card.jsx

```diff
@@ -0,0 +1,99 @@
+import { numberFormat } from '@automattic/jetpack-components';
+import { Card } from '@wordpress/components';
+import { arrowDown, arrowUp, Icon } from '@wordpress/icons';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './style.module.scss';
+
+const subtract = ( a, b ) => {
+	if ( typeof a !== 'number' || typeof b !== 'number' ) {
+		return null;
+	}
+
+	return a - b;
+};
+
+export const percentCalculator = ( part, whole ) => {
+	if ( typeof part !== 'number' || typeof whole !== 'number' ) {
+		return null;
+	}
```

Comment:
I see this being used twice in this file, could we move this to a `validateNumber` function or something like that? Or is there a function like it that already exists somewhere else in the plugin?

Reply:
This is a very personal opinion, but I don't usually like to abstract such things, considering that we would pretty much abstract it to a function that its only responsibility is to do exactly the same thing as the raw operation is doing: checking if a variable is a number.

Or you mean abstracting checking n parameters at once? Something like:
```js
const validateNumber = ( ...numbers ) => {
    return numbers.every( number => typeof number === 'number' );
}
```

Either way, that would add a tiny level of complexity that I personally don't think is necessary.